app/javascript/forms/form-buttons-redux.jsx

```diff
@@ -1,13 +1,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { combineReducers } from 'redux';
 
 import FormButtons from './form-buttons';
 
-function FormButtonsRedux(props) {
-  initReducer();
+function initReducer() {
+  if (initReducer.done) {
```

Comment:
You can also drop this bit now.

(Loading the module multiple times (assuming the same webpack runtime) will not cause `initReducer` to be called multiple times.)